templates/program/nph/order/sample-finalize.html.twig

```diff
@@ -72,7 +81,7 @@
                         <strong class="warning-text">
                             Finalized
                             by Biobank user
-                            on {{ sample.finalizedTs|date('n/j/Y g:ia', app.user.timezone) }}
+                            on {{ macros.displayDateAndTimezone(sample.finalizedTs, sample.finalizedTimezoneId) }}
```

Comment:
I see that for biobank view you are displaying timezone only if the order is finalized by biobank user. I think we need to display timezone for other orders too if it's biobank view.